.travis.yml

```diff
@@ -2,6 +2,6 @@ sudo: false
 cache: bundler
 language: ruby
 rvm:
-  - 2.4.1
+  - 3.0.5
```

Comment:
Shall I migrate this to Actions?

Reply:
It'd probably be good to migrate it to Actions! :)